.clang-format

```diff
@@ -0,0 +1,65 @@
+---
+# Language
+Language:        Cpp
+Standard:        Cpp11
+
+# Identation
+TabWidth:        4
+UseTab:          Always
+IndentWidth:     4
+ColumnLimit:     80
+
+# Identation detail
+AlignAfterOpenBracket: DontAlign
+ContinuationIndentWidth: 4
+BreakConstructorInitializers: AfterColon
+ConstructorInitializerIndentWidth: 4
+ConstructorInitializerAllOnOneLineOrOnePerLine: false
+BinPackParameters: true
+BinPackArguments: true
+AlignOperands: false
+AlignEscapedNewlines: DontAlign
+AccessModifierOffset: -4
+
+# Includes
+IncludeBlocks:   Regroup
+IncludeCategories:
+  # the "main header" implicitly gets priority 0
+  # system headers
+  - Regex:           '^<[^>]+>$'
+    Priority:        1
+  # non-system headers
+  - Regex:           '.*'
+    Priority:        2
+SortIncludes:    true
+
+# Spaces
+SpaceBeforeAssignmentOperators: true
+SpaceBeforeParens: ControlStatements
```

Comment:
I would change the key here to `Never`, as `ControlStatements` keywords are generally syntax highlighted and easily distinguishable without a space before them.

Reply:
I could agree, but our coding conventions enforce this. This PR only tries to follow the coding conventions.

You'd have to ask in discord (or an issue) to change the coding conventions first.

Reply:
I see this convention follows the Qt coding style, so I'm fine with it as is.